websites/B/BoxSeries/presence.ts

```diff
@@ -0,0 +1,160 @@
+	const [logo, cover, privacy, buttons, timestamps] = await Promise.all([
+			presence.getSetting<number>("logo"),
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("timestamps")
+		]),
+		titleMain =
+			document.querySelector("#dt_contenedor > div.module > div > div > h1")
+				?.textContent ?? "Pesquisa não encontrada",
+		titleEp =
+			document.querySelector("#single > h1.epih1")?.textContent.split(" - ") ??
+			"Pesquisa não encontrada",
+		pathArray = document.location.toString().split("/"),
+		presenceData: PresenceData = {
+			largeImageKey: ["default", "default_horizontal_tp"][logo],
+			startTimestamp: browsingTimestamp
+		};
+	if (new URLSearchParams(location.search).has("s")) {
+		presenceData.details = "Pesquisando po";
+		presenceData.state = `${
+			document.querySelector(
+				"#dt_contenedor > div.module > div > div > div > header > h1 > span"
+			)?.textContent ?? "Pesquisa não encontrada"
+		}`;
+		presenceData.smallImageKey = "search";
+	} else {
+		switch (pathArray[3]) {
+			case "calendario":
+				presenceData.details = "Vendo calendário do dia";
+				presenceData.smallImageKey = "reading";
+				break;
+			case "generos":
+				presenceData.details = "Vendo gêneros";
+				if (!privacy && pathArray[4]) presenceData.state = titleMain;
+				presenceData.smallImageKey = "reading";
+				break;
+			case "anos":
+				presenceData.details = "Vendo anos";
+				presenceData.smallImageKey = "reading";
+				break;
+			case "release":
+				presenceData.details = "Vendo liberação";
+				if (!privacy && pathArray[4]) presenceData.state = titleMain;
+				presenceData.smallImageKey = "reading";
+				break;
+			case "series":
+				presenceData.details = "Vendo séries";
+				if (cover) {
+					presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+						"div > div > div > div.poster-season > img"
+					).src;
+				}
+				if (!privacy && pathArray[4]) {
+					presenceData.state = `${
+						document.querySelector("div > div > div > div.data > h1")
+							?.textContent ?? "pesquisa não encontrada"
+					}`;
+					presenceData.smallImageKey = "reading";
+				}
+				break;
+			case "episodio":
+				if (cover) {
+					presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+						"#dt_contenedor > div.season-bg-bs > div > a > img"
+					).src;
+				}
+				presenceData.details = `${titleEp[0]}`;
+				presenceData.state = `${titleEp[1]}`;
+				presenceData.smallImageKey = video.paused ? "pause" : "play";
+				presenceData.smallImageText = video.paused
+					? (await strings).pause
+					: (await strings).play;
+				if (!video.paused) {
+					[, presenceData.endTimestamp] = presence.getTimestamps(
+						Math.floor(video.current),
+						Math.floor(video.duration)
+					);
+				}
+				if (buttons) {
+					presenceData.buttons = [
+						{
+							label: "Assistindo episódio",
```

Comment:
Lembra-te de que isto é um botão
```suggestion
							label: "Assistir episódio",
```